multi_column_editor/__init__.py

```diff
@@ -9,11 +9,15 @@
 import aqt.editor
 
 # A sensible maximum number of columns we are able to set
-MAX_COLUMNS = 18
 
-# Settings key to remember column count
-CONF_KEY_COLUMN_COUNT = 'multi_column_count'
 
+# Settings key to remember column count
+config= mw.addonManager.getConfig(__name__)
+if config is None:
+    config=dict()
+def getConfig(self,key,defaultValue=None):
+    transferConfig(self)
```

Comment:
I don't like that this happens on every invocation. It should happen only once on startup. I don't even mind removing it completely since it's only a minor inconvenience.

Reply:
I assume you are not talking about the four lines. 
the test on «config is None» is done only when the add-on is loaded, so once each time anki is loaded. 
The function transferConfig ensure that tre transfer is not done wice, sunig variable "transfer done". The test may be done before the function call, it may be more efficient (alas, anki has no python efficiency package in its environment, so I can't test)

I fear that, if the current configuration is lost, it will upset a lot of current user of your add-on. I could also try to tell the add-on to edit its own source, so that this function literally disapear after the first execution. It may actually be fun to do, but not necessarily wise

Reply:
It's fine, don't worry about it. You're probably right about bothering users so let's keep it.